manifests/e/Element/Element/1.11.96/Element.Element.installer.yaml

```
@@ -23,5 +23,8 @@ Installers:
 - Architecture: x64
   InstallerUrl: https://packages.element.io/desktop/install/win32/x64/Element%20Setup%201.11.96.exe
   InstallerSha256: 2F799AA8F5EDF39B184E548B60AE55B3E6E9BB305DF13BB0AD702CCB7863D812
+- Architecture: arm64
+  InstallerUrl: https://packages.element.io/desktop/install/win32/arm64/Element%20Setup.exe

```

```suggestion
  InstallerUrl: https://packages.element.io/desktop/install/win32/arm64/Element%20Setup%201.11.96.exe
```
It's best to use a version specific download link as then the manifest will still work even when a new version is released.

---

Thanks for the suggestions @a-mnich 

PR updated